### ISSUE TITLE
"on_on_completed" misprint

### ISSUE THREAD

**Issue by pzhin** [ReactiveX/RxRuby#93]
Hello, I think, that I found misprint
https://github.com/ReactiveX/RxRuby/blob/768c0845d307481aadfbacb3dda8a98619e1fe7e/lib/rx/operators/time.rb#L108
It seems like "on_on_completed" should be "on_completed"
So in my code I have such error
```
NoMethodError:         NoMethodError: undefined method 'on_on_completed' for #<Rx::Subject:0x0000000156fc10>
        Did you mean?  on_completed
            /home/pnevezhin/.rvm/gems/ruby-2.3.0/gems/rx-0.0.3/lib/rx/operators/time.rb:108:in 'block (5 levels) in window_with_time'
            /home/pnevezhin/.rvm/gems/ruby-2.3.0/gems/rx-0.0.3/lib/rx/operators/time.rb:108:in 'each'
            /home/pnevezhin/.rvm/gems/ruby-2.3.0/gems/rx-0.0.3/lib/rx/operators/time.rb:108:in 'block (4 levels) in window_with_time'
            /home/pnevezhin/.rvm/gems/ruby-2.3.0/gems/rx-0.0.3/lib/rx/operators/time.rb:107:in 'synchronize'
            /home/pnevezhin/.rvm/gems/ruby-2.3.0/gems/rx-0.0.3/lib/rx/operators/time.rb:107:in 'block (3 levels) in window_with_time'
            /home/pnevezhin/.rvm/gems/ruby-2.3.0/gems/rx-0.0.3/lib/rx/core/observer.rb:106:in 'on_completed'
            /home/pnevezhin/.rvm/gems/ruby-2.3.0/gems/rx-0.0.3/lib/rx/core/auto_detach_observer.rb:30:in 'on_completed_core'
            /home/pnevezhin/.rvm/gems/ruby-2.3.0/gems/rx-0.0.3/lib/rx/core/observer.rb:106:in 'on_completed'
            /home/pnevezhin/.rvm/gems/ruby-2.3.0/gems/rx-0.0.3/lib/rx/core/observer.rb:106:in 'on_completed'
            /home/pnevezhin/.rvm/gems/ruby-2.3.0/gems/rx-0.0.3/lib/rx/core/auto_detach_observer.rb:30:in 'on_completed_core'
            /home/pnevezhin/.rvm/gems/ruby-2.3.0/gems/rx-0.0.3/lib/rx/core/observer.rb:106:in 'on_completed'
            /home/pnevezhin/.rvm/gems/ruby-2.3.0/gems/rx-0.0.3/lib/rx/subjects/subject.rb:43:in 'block in on_completed'
            /home/pnevezhin/.rvm/gems/ruby-2.3.0/gems/rx-0.0.3/lib/rx/subjects/subject.rb:43:in 'each'
            /home/pnevezhin/.rvm/gems/ruby-2.3.0/gems/rx-0.0.3/lib/rx/subjects/subject.rb:43:in 'on_completed'
            /home/pnevezhin/.rvm/gems/ruby-2.3.0/gems/rx-0.0.3/lib/rx/core/auto_detach_observer.rb:30:in 'on_completed_core'
            /home/pnevezhin/.rvm/gems/ruby-2.3.0/gems/rx-0.0.3/lib/rx/core/observer.rb:106:in 'on_completed'
            /home/pnevezhin/.rvm/gems/ruby-2.3.0/gems/rx-0.0.3/lib/rx/core/auto_detach_observer.rb:30:in 'on_completed_core'
            /home/pnevezhin/.rvm/gems/ruby-2.3.0/gems/rx-0.0.3/lib/rx/core/observer.rb:106:in 'on_completed'
            /home/pnevezhin/.rvm/gems/ruby-2.3.0/gems/rx-0.0.3/lib/rx/core/observer.rb:106:in 'on_completed'
            /home/pnevezhin/.rvm/gems/ruby-2.3.0/gems/rx-0.0.3/lib/rx/core/auto_detach_observer.rb:30:in 'on_completed_core'
            /home/pnevezhin/.rvm/gems/ruby-2.3.0/gems/rx-0.0.3/lib/rx/core/observer.rb:106:in 'on_completed'
            /home/pnevezhin/.rvm/gems/ruby-2.3.0/gems/rx-0.0.3/lib/rx/operators/creation.rb:85:in 'block (2 levels) in generate'
            /home/pnevezhin/.rvm/gems/ruby-2.3.0/gems/rx-0.0.3/lib/rx/concurrency/scheduler.rb:36:in 'block in schedule_recursive'
            /home/pnevezhin/.rvm/gems/ruby-2.3.0/gems/rx-0.0.3/lib/rx/concurrency/scheduler.rb:96:in 'block in invoke_recursive'
            /home/pnevezhin/.rvm/gems/ruby-2.3.0/gems/rx-0.0.3/lib/rx/concurrency/scheduler.rb:109:in 'block (3 levels) in invoke_recursive'
            /home/pnevezhin/.rvm/gems/ruby-2.3.0/gems/rx-0.0.3/lib/rx/concurrency/scheduled_item.rb:29:in 'invoke'
            /home/pnevezhin/.rvm/gems/ruby-2.3.0/gems/rx-0.0.3/lib/rx/concurrency/current_thread_scheduler.rb:67:in 'run_trampoline'
            /home/pnevezhin/.rvm/gems/ruby-2.3.0/gems/rx-0.0.3/lib/rx/concurrency/current_thread_scheduler.rb:40:in 'schedule_relative_with_state'
            /home/pnevezhin/.rvm/gems/ruby-2.3.0/gems/rx-0.0.3/lib/rx/concurrency/local_scheduler.rb:20:in 'schedule_with_state'
            /home/pnevezhin/.rvm/gems/ruby-2.3.0/gems/rx-0.0.3/lib/rx/core/observable.rb:42:in '_subscribe'
            /home/pnevezhin/.rvm/gems/ruby-2.3.0/gems/rx-0.0.3/lib/rx/core/observable.rb:22:in 'subscribe'
            /home/pnevezhin/.rvm/gems/ruby-2.3.0/gems/rx-0.0.3/lib/rx/linq/connectable_observable.rb:15:in 'connect'
```

**Comment by pnevezhin** [comment]
@mattpodwysocki, excuse me, can you help me with this?

**Comment by csjune** [comment]
@pzhin Thank you for taking the time to report this bug. I'll fix it.

**Comment by csjune** [comment]
@pzhin Fixed on #103
